packages/oui-datagrid/src/datagrid.directive.js

```diff
@@ -16,6 +16,7 @@ export default () => {
             rows: "<?",
             rowsLoader: "&?",
             rowLoader: "&?",
+            subrows: "@?",
```

Comment:
No default value for this attribute (eg. `children`) ?
What do you think of `subrows-property` for the attribute name ? it's more explicit and follow the logic of `oui-column`.